recipes/mo-catnip/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mo-catnip" %}
+{% set version = "0.0.3" %}
```

Comment:
latest version on pypi looks to be `0.0.4`. Are you purposefully starting at an earlier version?

Reply:
changes the version name